src/SurveyRepo.js

```diff
@@ -1,8 +1,41 @@
 class SurveyRepo {
-  constructor() {
-    this.surveys = [];
-    this.answers = [];
+  constructor(dataFilePath) {
+    this.surveys = [...dataFilePath.surveys];
+    this.answers = [...dataFilePath.answers];
+    this.currentSurvey;
+    this.questionAndAnswers;
+
+  }
+
+  randomizeSurveys() {
+  	const randomize = function(surveys) {
+  		let currentIndex = surveys.length;
+  		let tempValue;
+  		let randomIndex;
+
+  		while(0 !== currentIndex) {
+  			randomIndex = Math.floor(Math.random() * currentIndex);
+  			currentIndex -= 1;
+
+  			tempValue = surveys[currentIndex];
+  			surveys[currentIndex] = surveys[randomIndex];
+  			surveys[randomIndex] = tempValue;
```

Comment:
With swapping values in this variable - we could refactor this to take advantage of array destructuring.